plugins/module_utils/config_loader.py

```
@@ -44,6 +39,17 @@ def __init__(self, config_file=None, warn_callback=None):
 
     def _load_config_file(self, config_file):
         """Loads and validates the configuration from the provided file."""
+        if PYYAML_IMPORT_ERROR:
+            raise ImportError(
+                "The 'pyyaml' package is required when using a config file. "
+                "Install it with: pip install pyyaml"
+            ) from PYYAML_IMPORT_ERROR
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)